src/tingle.css

```diff
@@ -21,7 +21,7 @@
   background: rgba(0, 0, 0, .8);
   opacity: 0;
   cursor: pointer;
-  transition: opacity .2s ease;
+  transition: all .2s ease;
```

Comment:
"all" causes a strange animation on open (due to the padding-top for overflow positioning)
 
[http://www.giphy.com/gifs/26FLjY68ivTAeRYBy](http://www.giphy.com/gifs/26FLjY68ivTAeRYBy)